api-report/runtime-definitions.api.md

```diff
@@ -115,7 +115,7 @@ export interface IContainerRuntimeBaseEvents extends IEvent {
 }
 
 // @public
-export interface IDataStore extends IFluidRouter {
+export interface IDataStore extends IFluidDataStoreRuntimeEntrypoint {
```

Comment:
I think you should add methods to this interface, instead of adding new interfaces.

Reply:
see: https://github.com/microsoft/FluidFramework/pull/11887#discussion_r966540316

Reply:
we could just implement Partial<IProvideFluidHandle> here, and not need a new interface

Reply:
same here, just add `Partial<IProvideFluidHandle>` to the implementation, but leave the interface unchanged for now